application/modules/invoices/controllers/Invoices.php

```
@@ -370,14 +370,14 @@ public function send_to_peppol($invoice_id): void
         $this->load->model('integrations/mdl_integrations');
         $this->load->library('crypt');
 
-        $settingsService = new App\Services\Integrations\IntegrationSettingsService(
+        $settingsService = new Core\Services\Integrations\IntegrationSettingsService(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '373', column '32'). (undefined)

(MissingImport)